app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args, **kwargs) -> any:
```

Comment:
The return type hint `any` is not specific. Consider using `Callable` or the actual return type of the function being decorated if known.